db/seeds.rb

```diff
@@ -59,6 +60,23 @@
 bridge_builders2 = Tenant.find(2)
 bridge_builders3 = Tenant.find(3)
 
+# user as a tenant
+tenant_user = User.create!(
+  :username => "Jwan622",
+  :first_name => "Jeff",
+  :last_name => "Wan",
+  :password => "password",
+  :email => "Jwan622@gmail.com",
+  :street => "31 Hillwood Court",
+  :city => "Staten Island",
+  :state => "NY",
+  :zipcode => 10305,
+  :country => "USA"
+)
+tenant_user.tenant = Tenant.create!(:location => "Shanghai",
+               :organization => "Shanghai Water Filterers"
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.